required/latex-lab/latex-lab-math.dtx

```diff
@@ -247,8 +247,15 @@
 %  It can be suppressed by an option with
 %  \texttt{math/tex/AF=false}, see below. 
```

Comment:
@u-fischer if I want to correct or improve the generated mathml , eg add `intent` attributes to improve generated text is the recommeded way to use `false` here and copy the `-luamml-mathml.html` file to `mathml.html` so it gets picked up by the existing mechanism? or is there a key specifically to disable the file being re-written but still allow its use.

Longer term one may hope to add such attributes via optional arguments in the tex source, but while testing, remediating the generated mathml is likely needed,

Reply:
@davidcarlisle yes the current idea is that you copy the file (or parts of it) into the mathml.html. You can also use some other name and then set the `math/mathml/sources={file1,file2}` (without extensions). You can disable the writing of the luamml with `math/mathml/luamml=false`, the existing file is still read. But imho adapting the luamml-file is rather fragile if you extend your document you can not reenable the automation.